README.md

```diff
@@ -12,17 +12,30 @@ Contains the __@itwin/insights-client__ package that wraps sending requests to t
 
```

Comment:
Update the `About this Repository` section. This package sends requests to the Reporting service, not to the project service.

Reply:
changed project to reporting